data/brands/amenity/fast_food.json

```diff
@@ -6380,6 +6380,7 @@
         "brand": "Евро Кебаб",
         "brand:en": "Еuro Кеbаb",
         "brand:ru": "Евро Кебаб",
+        "brand:wikidata": "109943635",
```

Comment:
Don't forget the `Q`  😄 
 
```suggestion
        "brand:wikidata": "Q109943635",
```

Reply:
> 

I'm not a real developer, I'm just pretending)

Reply:
no worries, computers are all pretend anyway